lib/factory/filter_configuration_builder.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:io';
```

Comment:
This file was merged in previous PR, remove it from this one

Reply:
File was deleted from this PR